services/ops/TopicWatcher/topic_watcher/agent.py

```diff
@@ -61,7 +62,44 @@ def __init__(self, config_path, **kwargs):
         self.config = utils.load_config(config_path)
         self.group_agent = {}
         self._connection = None
-
+        self.publish_settings = self.config.get('publish-settings')
+        self._remote_agent = None
```

Comment:
so we can have only one remote agent?

Reply:
Are you asking if we should allow the sending to more than one instance?

Reply:
yes

Reply:
I don't think so in this case...though that's an interesting thought.  We are just sending it from devices to a vc instance where an emailer will send it off to the correct party.